2020-08-05 14:59:34,565 -- CRITICAL -- в файле config_server_log.py -- в строке 22 -- Критическая ошибка
2020-08-05 14:59:34,565 -- ERROR -- в файле config_server_log.py -- в строке 23 -- Ошибка
2020-08-05 14:59:34,565 -- DEBUG -- в файле config_server_log.py -- в строке 24 -- Отладочная информация
2020-08-05 14:59:34,565 -- INFO -- в файле config_server_log.py -- в строке 25 -- Информационное сообщение
2020-08-05 15:49:50,749 -- DEBUG -- в файле server.py -- в строке 70 -- Сервер слушает все адреса
2020-08-05 15:49:50,750 -- DEBUG -- в файле server.py -- в строке 82 -- Сокет подготовлен
2020-08-05 15:49:50,751 -- DEBUG -- в файле server.py -- в строке 87 -- Слушаем порт
2020-08-05 15:49:54,219 -- DEBUG -- в файле server.py -- в строке 24 -- Принято сообщение от клиента: {'action': 'presence', 'time': 1596635394.2198038, 'user': {'account_name': 'Guest'}}
2020-08-05 15:49:54,219 -- DEBUG -- в файле server.py -- в строке 28 -- Сообщение {'action': 'presence', 'time': 1596635394.2198038, 'user': {'account_name': 'Guest'}} допустимо
2020-08-05 15:52:07,303 -- DEBUG -- в файле server.py -- в строке 24 -- Принято сообщение от клиента: {'action': 'presence', 'time': 1596635527.3033679, 'user': {'account_name': 'Guest'}}
2020-08-05 15:52:07,303 -- DEBUG -- в файле server.py -- в строке 28 -- Сообщение {'action': 'presence', 'time': 1596635527.3033679, 'user': {'account_name': 'Guest'}} допустимо
2020-08-05 15:55:20,026 -- CRITICAL -- в файле config_server_log.py -- в строке 22 -- Критическая ошибка
2020-08-05 15:55:20,026 -- ERROR -- в файле config_server_log.py -- в строке 23 -- Ошибка
2020-08-05 15:55:20,026 -- DEBUG -- в файле config_server_log.py -- в строке 24 -- Отладочная информация
2020-08-05 15:55:20,026 -- INFO -- в файле config_server_log.py -- в строке 25 -- Информационное сообщение
2020-08-05 15:55:39,602 -- DEBUG -- в файле server.py -- в строке 24 -- Принято сообщение от клиента: {'action': 'presence', 'time': 1596635739.6028955, 'user': {'account_name': 'Guest'}}
2020-08-05 15:55:39,603 -- DEBUG -- в файле server.py -- в строке 28 -- Сообщение {'action': 'presence', 'time': 1596635739.6028955, 'user': {'account_name': 'Guest'}} допустимо
2020-08-05 15:58:38,058 -- DEBUG -- в файле server.py -- в строке 24 -- Принято сообщение от клиента: {'action': 'presence', 'time': 1596635918.058995, 'user': {'account_name': 'Guest'}}
2020-08-05 15:58:38,058 -- DEBUG -- в файле server.py -- в строке 28 -- Сообщение {'action': 'presence', 'time': 1596635918.058995, 'user': {'account_name': 'Guest'}} допустимо
